wrapanapi/systems/ec2.py

```diff
@@ -268,6 +268,14 @@ def creation_time(self):
         self.refresh()
         return self.raw.creation_time
 
+    @property
+    def status_active(self):
+        self.refresh()
+        if self.raw.stack_status in StackStates.ACTIVE:
```

Comment:
This can be flattened to `return self.raw.stack_status in StackStates.ACTIVE`

Reply:
Done